.envrc

```diff
@@ -1,38 +1,2 @@
-PATH_DATA='/mnt/c/Users/Eren_Kiliclar/AppData/Local/Programs/Microsoft VS Code/data/'
-LOCAL_REGISTRY_PATH=models
 
```

Comment:
As Quentin and me put dotenv directly in line 1 it would be great if you could put dotenv in the first line so that we don't get any conflicts when merging the different branches.

Reply:
you're referring to the red zone removed. The updated .envrc file contains dotenv only, which is marked green at the rock bottom.

Reply:
Yeah I have seen that. But in the final version it will be in line 2 and not line one. I don't know if gh will complain when Quentin and I will merge our branches to master and have it in line 1.

Reply:
my dotenv is on the second row too.
![dotenv](https://github.com/michafdlr/AI_written_text_identifier/assets/106864913/13e96522-4642-49b8-9e5a-a3916f8547d0)
4d82e9)

Reply:
I think we talked past each other :) Changed it directly in the file.